storage/libradb/src/ledger_store/ledger_info_test.rs

```diff
@@ -29,11 +29,11 @@ prop_compose! {
                 let ledger_info = p.ledger_info();
                 LedgerInfoWithSignatures::new(
                     LedgerInfo::new(
-                        start_version + i as u64,
+                        start_epoch + i as Version,
```

Comment:
`as Version` ?
![billo](https://user-images.githubusercontent.com/22089200/63557010-c0ae6200-c4fb-11e9-8f6b-9cf692760295.jpg)

Reply:
The first param is `version` so I just assign `epoch_num` as `version`

Reply:
Why not pass both start_epoch and start_version ?

Reply:
The version cannot be random. If version_a < version_b then epoch_a must be < apoch_b. Assign proper random versions don't help what we are testing here...